srcpkgs/delta/patches/downgrade-git2.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/Cargo.toml b/Cargo.toml
-index 8c0c6d8..4fdaa9d 100644
+index d55c68e..b80247b 100644
 --- a/Cargo.toml
 +++ b/Cargo.toml
-@@ -46,7 +46,7 @@ vte = "0.11.0"
- xdg = "2.4.1"
+@@ -57,7 +57,7 @@ clap_complete = "4.4.4"
+ terminal-colorsaurus = "0.3.1"
  
  [dependencies.git2]
--version = "0.16.1"
+-version = "0.18.2"
```

Comment:
Why do we need to downgrade `git2`?

Reply:
It sadly pins the version of libgit2 used, so this is the only way to link to version 1.6 void provides.